R/games.R

```diff
@@ -1124,16 +1123,16 @@ sample_pa_age <- function(n, pa.exp, aging.exp, m = NULL, aging.bin = 300,
     m <- NULL
   }
   if (!is.null(out.seq) && length(out.seq) != n) {
-    stop("`out.seq' should be of length `n'")
+    cli::cli_abort("`out.seq' should be of length `n'")
```

Comment:
```suggestion
    cli::cli_abort("{.arg out.seq} must have length {.val n}, not {.val {length( out.seq)}}.'")
```

also following https://style.tidyverse.org/errors.html#problem-statement